modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavaVertXWebServerCodegen.java

```diff
@@ -0,0 +1,162 @@
+        supportingFiles.clear();
+        supportingFiles.add(new SupportingFile("supportFiles/openapi.mustache", resourceFolder, "openapi.yaml"));
+        supportingFiles.add(new SupportingFile("supportFiles/HttpServerVerticle.mustache", sourcePackageFolder, "HttpServerVerticle.java"));
+        supportingFiles.add(new SupportingFile("supportFiles/MainVerticle.mustache", sourcePackageFolder, "MainVerticle.java"));
+        supportingFiles.add(new SupportingFile("supportFiles/ApiResponse.mustache", sourcePackageFolder, "ApiResponse.java"));
+        supportingFiles.add(new SupportingFile("supportFiles/ApiException.mustache", sourcePackageFolder, "ApiException.java"));
+        supportingFiles.add(new SupportingFile("supportFiles/ParameterCast.mustache", sourcePackageFolder, "ParameterCast.java"));
+        supportingFiles.add(new SupportingFile("supportFiles/pom.mustache", "", "pom.xml"));
+
+        writeOptional(outputFolder, new SupportingFile("README.mustache", "", "README.md"));
+    }
+
+    @Override
+    public void postProcessModelProperty(CodegenModel model, CodegenProperty property) {
+        super.postProcessModelProperty(model, property);
+        if (!model.isEnum) {
+            model.imports.add("JsonInclude");
+            model.imports.add("JsonProperty");
+            if (model.hasEnums) {
+                model.imports.add("JsonValue");
+            }
+        }
+    }
+
+    @Override
+    public Map<String, Object> postProcessOperationsWithModels(Map<String, Object> objs, List<Object> allModels) {
+        Map<String, Object> newObjs = super.postProcessOperationsWithModels(objs, allModels);
+        Map<String, Object> operations = (Map<String, Object>) newObjs.get("operations");
+        if (operations != null) {
+            List<CodegenOperation> ops = (List<CodegenOperation>) operations.get("operation");
+            for (CodegenOperation operation : ops) {
+                operation.httpMethod = operation.httpMethod.toLowerCase(Locale.ROOT);
+
+                if (operation.returnType == null) {
+                    operation.returnType = "Void";
+                }
+            }
+        }
+        return newObjs;
+    }
+
+    @Override
+    public Map<String, Object> postProcessSupportingFileData(Map<String, Object> objs) {
+        generateYAMLSpecFile(objs);
+        return super.postProcessSupportingFileData(objs);
+    }
+
+    @Override
+    public CodegenOperation fromOperation(String path, String httpMethod, Operation operation, List<Server> servers) {
+        CodegenOperation codegenOperation =
+                super.fromOperation(path, httpMethod, operation, servers);
+
+        for (CodegenParameter codegenParameter : codegenOperation.allParams) {
+            if (codegenParameter.dataType.equals("File")) {
+                codegenParameter.dataType = "FileUpload";
+                codegenOperation.imports.remove("File");
+                codegenOperation.imports.add("FileUpload");
```

Comment:
What about update the importMapping instead of overriding fromOperation?

Reply:
I think an even better solution is to override the typeMapping
 `typeMapping.put("file", "FileUpload");`